### PR TITLE
V4/fix/populate state

### PR DESCRIPTION
**Pull request by filafb**
#### What is the purpose of this pull request?

Upon saving the state, we capitalize its name. When comparing using the options values (not capitalized) in the `dropdown` component, the `select` tag cannot find a match between current value and option values. The added code capitalize the state names in the `RUS` configure file, allowing the values to match when visiting the form address.

I branched off of the current [v4.7.0](https://github.com/vtex/address-form/tree/v4.7.0) tag, since it has the base I needed to start the work - [this PR](https://github.com/vtex/address-form/pull/333), which it's not yet merged into 4.x. I can rebase it once it is merged.

#### What problem is this solving?

This code fixes an issue preventing the `state` field to be populate when user visits their address in the `RUS` store. 

#### How should this be manually tested?
In this [workspace](https://fixstate--ruwhirlpoolqa.myvtex.com), as a logged user, add or edit an address with a state value. When visiting the form again, the `state` field should be already populate with the value you added.

#### Screenshots or example usage

_Currently behavior_

before

_Fixed version_

fixed

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Comment by filafb** [comment]
@wdsrocha and @lucasecdb I added you both as reviewers here since you were the ones approving [this PR](https://github.com/vtex/address-form/pull/333). LMK if I should point it to a different person. Thanks!

**Comment by wdsrocha** [comment]
@filafb, can you explain why this mismatch between the dropdown and the value happens? I mean, where does the capitalized value is coming from? I've never seen this bug in this component for other idioms (maybe @darlenedms have seen it?), so I have the feeling that just capitalizing everything isn't the best approach.

Sure, the proposed fix does solve the problem, so if we can't think of a better solution, we can merge this.

File: react/country/RUS.js [hunk 1]
```diff
@@ -1,151 +1,148 @@
...
-  getOneLevel,
-  getTwoLevels,
-} from '../transforms/addressFieldsOptions.js'
+import { getOneLevel } from '../transforms/addressFieldsOptions.js'
 
-const countryData = {
+const countryDataRaw = {
   'Республика Адыгея': {
-    'Maykop': '385000',
+    Maykop: '385000',
```
Comment (wdsrocha) [inline react/country/RUS.js:7]: Could you undo this formatting change? In the following lines as well
```suggestion
    'Maykop': '385000',
```

Just so we can keep a more precise PR change diff, and won't pollute the git blame as well

**Comment by filafb** [comment]
> @filafb, can you explain why this mismatch between the dropdown and the value happens? I mean, where does the capitalized value is coming from? I've never seen this bug in this component for other idioms (maybe @darlenedms have seen it?), so I have the feeling that just capitalizing everything isn't the best approach.
> 
> Sure, the proposed fix does solve the problem, so if we can't think of a better solution, we can merge this.

I haven't tracked down where we are capitalizing the state name - I know that wouldn't be a problem in Brazil because we have only the initials already capitalized, like MG and RN.

But you can see the where the mismatch happens by looking at the React dev tool. Below two screenshots:
1. Although the field value is `"МОСКОВСКАЯ ОБЛАСТЬ___101194"`, the state field - `Регион` - is empty. I guess this information is coming from an entry in MD or something;

Screen Shot 2021-05-07 at 3 07 47 PM

2. The list of options brings the value (which React uses to match the selected value) in the same way we enter them in the address form, in this case, not capitalized. This explain the mismatch.

Screen Shot 2021-05-07 at 3 08 06 PM

I agree this is not the scalable solution, as that might be affecting other countries, but I see no harm capitalizing the states name for RUS while we look for a better approach.

File: react/country/RUS.js (hunk 1 above)
Comment (filafb) [reply react/country/RUS.js:7]: I can do that, but this is prettier fixing the styles. I guess the previous PR haven't run `lint-staged` to match the prettier configuration. Let me know what you think.

Comment (wdsrocha) [reply react/country/RUS.js:7]: Ah, I see. In that case, there is no need to change. Thanks for linting it!

**Review by wdsrocha — APPROVED** [review]
Thanks for the screenshots, it's more clear now! If the next reviewer (probably on Monday) don't have another solution in mind, the PR LGTM

**Review by lucasecdb — APPROVED** [review]

**Pull request merged**